esp/templates/survey/review.tex

```diff
@@ -11,25 +11,37 @@
 
 \vspace{0.3in}
 
-{% if not surveys|length_is:0 %}
-Students from {{ prog.niceName }} filled out an online survey asking some questions about the program as a whole.  Their responses are summarized below.
-{% endif %}
-
 {% for s in surveys %}
 \vspace{0.25in}
 {% if not forloop.first %}\clearpage{% endif %}
     \begin{center}
-    \begin{longtable}{|l|l|} \hline
-    \multicolumn{2}{|c|}{\cellcolor{esphead} ~ } \\
-    \multicolumn{2}{|c|}{\cellcolor{esphead} \Large {Responses for {{ s.name }}} } \\ 
-    \multicolumn{2}{|c|}{\cellcolor{esphead} ~ } \\\hline
+    \begin{longtable}{|p{0.5\linewidth}|p{0.5\linewidth}|} \hline
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} \Large {Responses for {{ s.name }}} } \\ 
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\\hline
     {% for q in s.display_data.questions %}
         {% with s.num_participants as num_participants %}
             {% include "survey/answers.tex" %}
         {% endwith %}
     {% endfor %}
     \end{longtable}
     \end{center}
+    {% for c in s.perclass_data %}
+        \clearpage
+        \begin{center}
+        \begin{longtable}{|p{0.5\linewidth}|p{0.5\linewidth}|} \hline
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} \Large {Responses for {{ c.class.emailcode }}: {{ c.class.title|texescape }}} } \\ 
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\\hline
```

Comment:
Can we get the blue box to cover the entire width of the table (right now its a little short, width wise):
![image](https://user-images.githubusercontent.com/25392075/79692929-f4ca5600-8235-11ea-8cbb-7dfec1c2315e.png)

Reply:
Done!